bootstrap-extensions/src/main/java/de/agilecoders/wicket/extensions/markup/html/bootstrap/ladda/LaddaAjaxButton.java

```diff
@@ -115,7 +104,9 @@ protected void updateAjaxAttributes(AjaxRequestAttributes attributes) {
     @Override
     protected <L extends Serializable> Component newLabel(String markupId, IModel<L> model) {
         Component label = super.newLabel(markupId, model);
-        label.add(AttributeModifier.append("class", "ladda-label"));
```

Comment:
Why `ladda-label` is being replaced with `sr-only` ?
At https://msurguy.github.io/ladda-bootstrap/ I still see `ladda-label` being used. Or you use a newer Ladda implementation for Bootstrap 4.x ?

Reply:
Hello @martin-g, https://msurguy.github.io/ladda-bootstrap/ is clearly state `Ladda UI for Bootstrap 3`
I find it broken and tried to adapt to https://getbootstrap.com/docs/4.4/components/spinners/
I'll try to update javadocs and drop outdated links

Reply:
Don't waste your time with it!
I suggest to just remove Ladda from Bootstrap 4.x branch(es).
https://github.com/msurguy/ladda-bootstrap is abandoned since late 2017.

Reply:
I need spinner button
And it works now

I have dropped old ladda-button and make everything work with
Up-to-date spinner from bootstrap 4

Reply:
Awesome! Then maybe we should just rename the component & behavior!

Reply:
Sure
Will do it later today
Also will try to update javadocs